docs_user/modules/proc_adopting-networker-services-to-the-data-plane.adoc

```diff
@@ -214,24 +196,25 @@ endif::[]
         - openvswitch3.1
 EOF
 ----
++
+* If TLS Everywhere is enabled, change `spec:tlsEnabled` to `true`.
 
-. Ensure that the `ovn-controller` settings that are configured in the `OpenStackDataPlaneNodeSet` CR are the same as were set in the `Networker` nodes before adoption. This configuration is stored in the `external_ids`` column in the `Open_vSwitch` table in the Open vSwitch database:
+. Ensure that the `ovn-controller` settings that are configured in the `OpenStackDataPlaneNodeSet` CR are the same as the settings that were set in the `Networker` nodes before adoption. This configuration is stored in the `external_ids`` column in the `Open_vSwitch` table in the Open vSwitch database:
 +
 ----
 ovs-vsctl list Open .
 ...
 external_ids        : {hostname=standalone.localdomain, ovn-bridge=br-int, ovn-bridge-mappings=<bridge_mappings>, ovn-chassis-mac-mappings="datacentre:1e:0a:bb:e6:7c:ad", ovn-cms-options=enable-chassis-as-gw, ovn-encap-ip="172.19.0.100", ovn-encap-tos="0", ovn-encap-type=geneve, ovn-match-northd-version=False, ovn-monitor-all=True, ovn-ofctrl-wait-before-clear="8000", ovn-openflow-probe-interval="60", ovn-remote="tcp:ovsdbserver-sb.openstack.svc:6642", ovn-remote-probe-interval="60000", rundir="/var/run/openvswitch", system-id="2eec68e6-aa21-4c95-a868-31aeafc11736"}
 ...
 ----
-
 +
 * Replace `<bridge_mappings>` with the value of the bridge mappings in your configuration, for example, `"datacentre:br-ctlplane"`.
 
 . Optional: Enable `neutron-metadata` in the `OpenStackDataPlaneNodeSet` CR:
 +
 [source,yaml]
 ----
-oc patch openstackdataplanenodeset openstack-networker --type='json' --patch='[
```

Comment:
In steps 3 and 4, should users replace the following values with their own?
`openstackdataplanenodeset openstack-networker`

Reply:
they should replace `openstack-networker`